ui/app/manager/src/pages/page-account.ts

```diff
@@ -180,6 +184,7 @@ export class PageAccount extends Page<AppStateKeyed>  {
         }
 
         const readonly = !manager.hasRole(ClientRole.WRITE_ADMIN);
+        this._getPasswordPolicy();
```

Comment:
`render()` is a Lit function that gets called often during every state update,
Expensive functions like this `_getPasswordPolicy()` HTTP request should happen only once.
I would place this in a `connectedCallback()` function, which gets called before initial page render.
```typescript
connectedCallback() {
    super.connectedCallback();
    this._getPasswordPolicy().then(policy => {
        this._passwordPolicy = policy;
    })
}
```